user_guide/feature_contexts/hooking_into_the_test_process.rst

```diff
@@ -160,33 +162,39 @@ There are two suite hook types available:
 * ``@BeforeSuite`` - executed before any feature runs.
 * ``@AfterSuite`` - executed after all features have run.
 
-Feature Hooks
--------------
+Tagged Hooks
```

Comment:
Putting this section in the middle of the description of the different hook types does not make sense. This is about filtering when the hook should be applied, which is a different topic than describing the different types of hooks